diff --git a/Makefile.am b/Makefile.am
index 8e5f72b1..36cf360e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -11,7 +11,7 @@ dist_EXTRA_tmux_SOURCES = compat/*.[ch]
 # Preprocessor flags.
 AM_CPPFLAGS += @XOPEN_DEFINES@ \
 	-DTMUX_VERSION='"@VERSION@"' \
-	-DTMUX_CONF='"$(sysconfdir)/tmux.conf:~/.tmux.conf:$$XDG_CONFIG_HOME/tmux/tmux.conf:~/.config/tmux/tmux.conf"' \
+	-DTMUX_CONF='"@nixTmuxConf@"' \
 	-DTMUX_LOCK_CMD='"@DEFAULT_LOCK_CMD@"' \
 	-DTMUX_TERM='"@DEFAULT_TERM@"'
 
diff --git a/cfg.c b/cfg.c
index adac60e1..80f093e2 100644
--- a/cfg.c
+++ b/cfg.c
@@ -84,6 +84,11 @@ start_cfg(void)
 		cmdq_append(c, cfg_item);
 	}
 
+ 	printf("cfg_files:\n");
+	for (u_int i = 0; i < cfg_nfiles; i++) {
+		printf("%s\n", cfg_files[i]);
+	}
+
 	if (cfg_quiet)
 		flags = CMD_PARSE_QUIET;
 	for (i = 0; i < cfg_nfiles; i++)
@@ -96,6 +101,7 @@ int
 load_cfg(const char *path, struct client *c, struct cmdq_item *item,
     struct cmd_find_state *current, int flags, struct cmdq_item **new_item)
 {
+	printf("loaded file: %s\n", path);
 	FILE			*f;
 	struct cmd_parse_input	 pi;
 	struct cmd_parse_result	*pr;
diff --git a/tmux.c b/tmux.c
index a9619baf..9beac465 100644
--- a/tmux.c
+++ b/tmux.c
@@ -372,6 +372,8 @@ main(int argc, char **argv)
 	if (**argv == '-')
 		flags = CLIENT_LOGIN;
 
+ 	printf("TMUX_CONF: %s\n", TMUX_CONF);
+
 	global_environ = environ_create();
 	for (var = environ; *var != NULL; var++)
 		environ_put(global_environ, *var, 0);
@@ -379,6 +381,11 @@ main(int argc, char **argv)
 		environ_set(global_environ, "PWD", 0, "%s", cwd);
 	expand_paths(TMUX_CONF, &cfg_files, &cfg_nfiles, 1);
 
+ 	printf("cfg_files:\n");
+	for (u_int i = 0; i < cfg_nfiles; i++) {
+		printf("%s\n", cfg_files[i]);
+	}
+
 	while ((opt = getopt(argc, argv, "2c:CDdf:lL:NqS:T:uUvV")) != -1) {
 		switch (opt) {
 		case '2':
